apps/meteor/tests/e2e/federation/page-objects/fragments/home-sidenav.ts

```diff
@@ -61,9 +61,8 @@ export class FederationSidenav {
 		await this.page.locator('role=navigation >> role=button[name=Search]').click();
 		await this.page.locator('role=search >> role=searchbox').focus();
 		await this.page.locator('role=search >> role=searchbox').type(name);
-		await this.page.locator(`role=search >> role=listbox >> role=link >> text="${name}"`).first().waitFor();
-		await this.page.waitForTimeout(2000);
-		await this.page.locator(`role=search >> role=listbox >> role=link >> text="${name}"`).first().click();
+		await this.page.waitForTimeout(3000);
+		await this.page.locator(`role=search >> role=listbox >> role=link`).first().click();
```

Comment:
is `await this.page.locator(`role=search >> role=listbox >> role=link >> text="${name}"`).first().waitFor();` not enough? I can see why waiting after the element is displayed brings any value to the test
I'm against timeout in tests, usually introduces flaky tests